src/utils/transform.js

```diff
@@ -379,7 +395,8 @@ export function flatReactProp(reactProp, componentID) {
 export const getDependentQueries = (store, componentID, orderOfQueries = []) => {
 	const finalQuery = {};
 	const react = flatReactProp(store.dependencyTree[componentID], componentID);
-	react.forEach((component) => {
+	react.forEach((componentObject) => {
```

Comment:
@mohdashraf010897 You renamed the `component` to `componentObject` but didn't update all the instances (for e.g line no. 400). You can revert it back or update all the instances.

Reply:
@bietkul check line no. 399